presets/4.3/rates/Vanover.txt

```diff
@@ -0,0 +1,35 @@
+#$ TITLE: Vanover racing/freestyle rates
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: RATES
+#$ STATUS: COMMUNITY
+#$ KEYWORDS: racing, race, vanover, rates, MultiGP, freestyle
+#$ AUTHOR: Alex Vanover
+#$ DESCRIPTION: Racing and freestyle rates by Vanover.
+#$ DESCRIPTION: 488 deg/sec for all axes sounds slow, but it's not if you check Vanover's freestyle and race videos.
+#$ DESCRIPTION: Alex - 2017 MultiGP Champion and DRL pilot 2019, 2020, 2021. 2018 FAI Drone Sports and DR1 Champion. Professional acro drone operator for cinematrography.
+#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/96
+#$ INCLUDE: presets/4.3/rates/defaults.txt
+
+set rates_type = BETAFLIGHT
+set roll_rc_rate = 100
+set pitch_rc_rate = 100
+set yaw_rc_rate = 100
+set roll_expo = 1
+set pitch_expo = 1
+set yaw_expo = 1
```

Comment:
This is 10

Reply:
it is not

Reply:
https://rotorriot.com/products/vanovers-rates
![image](https://user-images.githubusercontent.com/31283897/143738054-ff2a2f56-1e5c-47fd-8178-0e1bc49c3000.png)

Reply:
I will double-check, but that's what he sent to me.

Reply:
At least *someone* flies with *my* standard race centre sensitivity :-)

Reply:
Confirmed with Vanover, rotor rioir rates are not correct.